vectordatasource/transform.py

```diff
@@ -3622,3 +3622,64 @@ def simplify_and_clip(ctx):
             simplified_features.append(simplified_feature)
 
         feature_layer['features'] = simplified_features
+
+
+def normalize_operator_values(shape, properties, fid, zoom):
+    """
+    There are many operator-related tags, including 'National Park Service',
+    'U.S. National Park Service', 'US National Park Service' etc that refer
+    to the same operator tag. This function promotes a normalized value
+    for all alternatives in specific operator values.
+
+    See https://github.com/tilezen/vector-datasource/issues/927.
+    """
+
+    operator = properties.get('operator', None)
+    if operator is not None:
+        if operator in ('National Park Service',
```

Comment:
Resolved in person, all of these are in the USA (and there are hundreds of them), so okay to leave as-is.